app/views/spree/admin/products/new.js.erb

```diff
@@ -1,5 +1,5 @@
 <%# This chunk is just a copy of Spree's core/app/views/spree/admin/products/new.js.erb %>
-$("#new_product").html('<%= escape_javascript(render :template => "spree/admin/products/new", :formats => [:html], :handlers => [:erb]) %>');
+$("#new_product").html('<%= escape_javascript(render :template => "spree/admin/products/new", :formats => [:html], :handlers => [:haml]) %>');
 handle_date_picker_fields();
 <% unless Rails.env.test? %>
   $('.select2').select2();
```

Comment:
can we convert this to haml?

Reply:
It's the same thing with autocomplete [in your comment](https://github.com/openfoodfoundation/openfoodnetwork/pull/2955#issuecomment-434363720), maybe I can open an issue for these .js.erb files that have to be converted and moved ?

Reply:
I had the same doubt but my concern is that it'll make it harder to compare it against Spree's one in the coming version. I guess however that if we convert OFN views to haml, we canvert this one too.

Reply:
yes, I think we need to get rid of erb completly.